### DISCUSSION TITLE
milvus is not able to get connected with database

### DISCUSSION THREAD

**Discussion by Anshuman-gitt** [milvus-io/milvus#31706] [Q&A and General discussion]
i have deployed the milvus on k8 ( using loadbalancer) on azure i have followed only step by step this doc https://milvus.io/docs/azure.md and when i connect my database with milvus it is showing an error of 
MilvusException: (code=2, message=Fail connecting to server on 4.246.41.153:19530. Timeout> Disconnected from Milvus
i have tried with other ports also same error is showing and when i ping on that external ip it is showing 100% packet loss
![Screenshot 2024-03-28 173725](https://github.com/milvus-io/milvus/assets/159427179/c80a7eac-5b20-4df9-9b35-f84daad5fcf0)

**Comment by yhmo** [comment]
Wait @LoveEachDay to help.

**Comment by LoveEachDay** [comment]
@Anshuman-gitt Could you try following steps to diagnose:

1.  run `kubectl port-forward pod/<milvus-proxy-pod> -n <namespace> 19530:19530`
2. try access `localhost:19530` with your application
3. if step 2 runs successfully, then milvus setup is all right, you'd check the load balancer provided by azure; if not, check whether all milvus pods and its dependencies are running well.
